### ISSUE TITLE
Completion after org-link gives org headings

### ISSUE THREAD

**Issue by OlMon** [minad/corfu#114]
Greetings,

As probably many I switched from company. I never noticed this behavior in company, but I am not sure if this has anything to do with company or corfu.

Trying to write something in a line with an Header Org-Link or specially  after the sequence "[[*" the completion presents the heading of the org files. It does not have to be right after the sequence the whole line triggers the behavior.

Thanks!

Cheers.

**Comment by minad** [comment]
Yes, this is a feature of the org capf backend. You can also observe it without Corfu if you use default completion.
